tests/Console/ApplicationTest.php

```diff
@@ -29,4 +29,9 @@ public function testApplication()
         $app = new Application();
         static::assertStringMatchesFormat('%s by <comment>Fabien Potencier</comment> and <comment>Dariusz Ruminski</comment>', $app->getLongVersion());
     }
+
+    public function testGetMajorVersion()
+    {
+        static::assertSame(2, Application::getMajorVersion());
```

Comment:
this will break on v3, maybe better to make it dynamic;

```
    public function testGetMajorVersion()
    {
        static::assertRegExp(sprintf('#^%d\.\d{1,2}\.\d{1,2}-[A-Z]+.*$#', Application::getMajorVersion()), Application::VERSION);
    }
```

Reply:
Meh, not worth it IMO, changing the number while merging into `3.x` will be fine.